### ISSUE TITLE
Label text is slightly covered in plot with renderImage

### ISSUE THREAD

**Issue by xdk78** [data-forge-notebook/plot#39]
After creating plot with `renderImage`  with config like this label `y` is slightly covered
```js
  const plt = plot(
    {
      x: x,
      y: y,
    },
    {
      x: {
        label: {
          font: {
            size: "18",
            family: "Verdana",
          },
          text: "Frequency [Hz]",
        },
      },
      y: {
        label: {
          font: {
            size: "18",
            family: "Verdana",
          },
          text: "Magnitude",
        },
      },
    }
  );
  ```
example plot
![c](https://user-images.githubusercontent.com/9966516/114053980-ba672800-988f-11eb-8f1e-8c978c9b2cb1.png)
Maybe some padding in https://github.com/data-forge-notebook/plot/blob/master/packages/render-image/template-src/template/assets/index.css will fix this?

**Comment by ashleydavis** [comment]
Can you please submit code so I can test this.

**Comment by xdk78** [comment]
I had to simplify code, so sorry for those very long arrays, but works the same as mine original code.
Just copy code from https://pastebin.com/e5WsQgFW and run with typescript

**Comment by ashleydavis** [comment]
Thanks for the example code.

Can I run that with pastebin? I didn't see any run button.

**Comment by xdk78** [comment]
No, just copy code from pastebin, create new file on your computer and run with Typescript compiler.

**Comment by ashleydavis** [comment]
If you could set it so I could just run it that would save me some time.

**Comment by xdk78** [comment]
Sorry but I don't know how to easily set up online version of node.js environment with Typescript

**Comment by xdk78** [comment]
I think it should be tested locally

**Comment by ashleydavis** [comment]
Yeah I'm happy to test it locally... if you could just provide me with a working test case please that will make it quick for me to replicate the problem.

You can email me a zip file to ashley@codecapers.com.au or you can use a GitHub repo, or you can attach a zip file here I think.

Thanks.
